src/Workspaces/CSharp/Portable/LanguageServices/CSharpSemanticFactsService.cs

```diff
@@ -371,5 +371,11 @@ private SymbolInfo GetSymbolInfo(SemanticModel semanticModel, SyntaxNode node, S
 
         public bool IsInsideNameOfExpression(SemanticModel semanticModel, SyntaxNode node, CancellationToken cancellationToken)
             => (node as ExpressionSyntax).IsInsideNameOfExpression(semanticModel, cancellationToken);
+
+        public bool CanConvert(SemanticModel semanticModel, SyntaxNode node, ITypeSymbol type)
```

Comment:
can you rename to something like `ImplicitConversionExists`.  'CanConvert' is very vague.